mkdocs.yml

```diff
@@ -153,6 +153,7 @@ nav:
     - Ports: "contributing/ports.md"
     - Cluster Addons & Manager : "contributing/addons.md"
   - Releases:
+    - "1.22": releases/1.22-NOTES.md
```

Comment:
This should not be added until 1.22 goes to Beta.

Reply:
Ah, my mistake. I didn't realize that was the policy.